public/data/itself13.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Ayush Singhal",
+    "bio": "MERN stack developer",
+    "avatar": "https://github.com/itself13.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itself13",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dx_singhal",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/ii_am_ayuuu/",
+        "icon": "Instagram"
+      }
+    ]
+}
```

Comment:
Missing an endline.
```suggestion
}

```

Reply:
Do not know why, but I requested the same changes: https://github.com/EddieHubCommunity/LinkFree/pull/677#discussion_r753069209 but this was resolved without commiting 🤔